main.py

```diff
@@ -3,4 +3,81 @@
 # 3. 계산기 코드 짜기
 # 4. 예외처리
 # 5. 예외처리
-# 6. 이스터에그
\ No newline at end of file
+# 6. 이스터에그
+ 
+
+class Stack():                      #Stack 클래스 생성
+    def __init__(self):             #클래스 초기화
+        self.stack = []
+    
+    def push(self, data):           #스택에 데이터 추가
+        self.stack.append(data)     
+    
+    def pop(self):                  #스택에서 데이터 반환
+        popObject = None
+        if self.isEmpty():         
+            print("Stack is Empty")
+        else:
+            popOjbect = self.stack.pop()
+        
+        return popObject
+    
+    def top(self):                  #스택의 가장 위에 있는 데이터 반환
+        topObject = None
+        if self.isEmpty():
+            print("Stack is Empty")
+        else:
+            topObject = self.stack[-1]
+        
+        return topObject 
+    
+    def isEmpty(self):              #스택이 비어있으면 True, 채워져있으면 False
+        isEmpty = False
+        if len(self.stack) == 0:
+            isEmpty = True
+        
+        return isEmpty
+
+
+
+class Node:                         #Linked list를 위한 Node 클래스 생성
+    def __init__(self, data):
+        self.data = data
+        self.next = None
+
+class linkedListStack():            #linkedListStack 클래스 생성
```

Comment:
```suggestion
class LinkedListStack():            #linkedListStack 클래스 생성
```
과 같이 클래스명은 첫문자가 대문자로 카멜케이스 맞춰주세요

Reply:
LinkedListStack 클래스는 지움으로 해결하였습니다.